lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -76,7 +76,7 @@
         % elif verification_status.get('status') == VERIFY_STATUS_APPROVED:
           <span class="sts-enrollment" title="${_("You're enrolled as a verified student")}">
             <span class="label">${_("Enrolled as: ")}</span>
-            <img class="deco-graphic" src="${static.url('images/verified-ribbon.png')}" alt="${_("ID Verified Ribbon/Badge")}" />
+            <img class="deco-graphic" src="${static.url('images/verified-ribbon.png')}" alt="" />
```

Comment:
@symbolist yes, its not important to include the alt attribute for the ribbon. All that is needed here is the accessible text below i.e "Verified Student"  otherwise the screen reader user hears "ID Verified Ribbon/Badge Verified Student" Including an empty alt attribute is the standard method for "hiding" an unnecessary image from a screen reader user.  What we *don't* want to do is omit the alt attribute altogether. Thanks for checking.